### ISSUE TITLE
[Bug] Rigidbodies ignore Position Constraint

### ISSUE THREAD

**Issue by Myterian** [FlaxEngine/FlaxEngine#2738]
**Issue description:**
Rigidbodies ignore Position Constraint, when a force is applied. This can be gravity or a custom force thru script.

**Steps to reproduce:**
1. Create a rigidbody with a collider child
2. Lock Position Y and set useGravity to true
3. Press play. It still falls

Alternatively
1. Create a rigidbody with a collider child
2. Lock Position
3. Create a script, that adds a forward force to the rigidbody
4. Press play. Rigidbody still moves

**Minimal reproduction project:**
Blank

**Flax version:**
1.8.6512.2

**Comment by Tryibion** [comment]
Can you change the physics solver type in Physics settings to TGS instead of PGS and test? I noticed that the version of PhysX that Flax uses has a bug that ignores position constraints while using the PGS solver.

**Comment by Myterian** [comment]
Hey, sorry for getting back so late, wasn't notified about your answer. I've tried to change the physics solver, but the problem still persists

**Comment by GasimoCodes** [comment]
Replicated in 1.9.6606. Using CCD solves the object ignoring collision when constraints are locked. When constraint is on the object still gets affected by gravity even when Constaints = Lock_All is set though.

**Comment by GasimoCodes** [comment]
Also same issue as mentioned in #2216. I suggest closing this thread and moving it over there or vice-versa. @Tryibion

**Comment by Myterian** [comment]
Re-read the issue description that I wrote. I can't remember why I used gravity as an example, but my problem is the position constraint does not work for any axis when any force is applied (not just gravity, but movement too). Probably should update this

**Comment by mafiesto4** [comment]
Yeah it looks like it's bug in PhysX: https://github.com/NVIDIAGameWorks/PhysX/issues/120
